### ISSUE TITLE
InstallCleanup doesn't specify format of version argument

### ISSUE THREAD

**Issue by x-cubed** [MicrosoftDocs/visualstudio-docs#7263]
On this page describing InstallCleanup.exe, there is no description of what suitable values for version are.

On my machine, I had VS2019 16.11.4 installed, alongside VS2022. VS2019 was unable to update, repair or uninstall as it couldn't find ML Model Builder. I wanted to remove VS2019 so that I could re-install it. When I tried running ``InstallCleanup.exe -i 2019`` and later ``InstallCleanup.exe -i 16.11.4``, the command completed without finding anything to delete.

When I ran InstallCleanup.exe without any arguments, expecting to see a list of known instances, it went ahead and deleted both 2019 and 2022 from my machine. The command listed instance names of ``d41eceae`` and ``d653e634``, so was that what the ``-i`` switch was expecting?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 2a3b0ef4-76d9-9c1e-4625-9cd56689b43a
* Version Independent ID: e882e8dc-8fb1-231d-d0bb-1f3e9dfec90d
* Content: [Remove Visual Studio](https://docs.microsoft.com/en-us/visualstudio/install/remove-visual-studio?view=vs-2022)
* Content Source: [docs/install/remove-visual-studio.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/install/remove-visual-studio.md)
* Product: **visual-studio-windows**
* Technology: **vs-installation**
* GitHub Login: @anandmeg
* Microsoft Alias: **meghaanand**

**Comment by WilliamAntonRohm** [comment]
@x-cubed -- Carey, thank you for your feedback.  Please consider these resources:
- [Visual Studio on Microsoft Q&A](https://docs.microsoft.com/answers/products/vs) 
- [Visual Studio MSDN Forum](https://social.msdn.microsoft.com/Forums/vstudio) 
- [Stack Overflow](https://stackoverflow.com/search?q=%22visual+studio%22)  
- [Reddit](https://www.reddit.com/r/VisualStudio)

@anandmeg -- Megha, please look into this issue.

**Comment by anandmeg** [comment]
@x-cubed, thank you for your feedback! We've updated the page: [Remove Visual Studio](https://review.docs.microsoft.com/en-us/visualstudio/install/remove-visual-studio?view=vs-2022&branch=pr-en-us-9150) to include an example of using the i switch. The changes should be live later today.

Thanks again for your contribution to making our docs better. Closing this now. Let us know if you have any other feedback.
